src/App/Game/Game.tsx

```diff
@@ -107,6 +107,14 @@ export default class Game extends React.Component<IGameProps, IGameState> {
     if (!this.gameDef) {
       return <MessagePageClass type={'error'} message={'Game Not Found'} />;
     }
+    const validGameModes: string[] = [];
```

Comment:
Replace this with an one liner:

const valisGameModes = this.gameDef.modes.map((m) => m.toLowerCase())